### PR TITLE
[FIX] purchase_ux: net_price in PO_line

### PR DESCRIPTION
**Pull request by mav-adhoc**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/purchase/pull/175).

File: purchase_ux/models/purchase_order_line.py [hunk 1]
```diff
@@ -247,3 +247,10 @@ def _compute_price_unit_and_date_planned_and_name(self):
                     price_unit = line.product_id.uom_id._compute_price(price_unit, line.product_uom)
                 line.price_unit = price_unit
         return res
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        lines = super(PurchaseOrderLine, self).create(vals_list)
+        for line in lines:
+            line._compute_price_unit_and_date_planned_and_name()
```
Comment (augusto-weiss) [inline purchase_ux/models/purchase_order_line.py]:
```
        lines._compute_price_unit_and_date_planned_and_name()
```

**Review by augusto-weiss — CHANGES_REQUESTED** [review]

**Comment by augusto-weiss** [comment]
@roboadhoc r+

**Comment by fw-bot-adhoc** [comment]
@mav-adhoc @augusto-weiss this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/purchase#176

**Comment by fw-bot-adhoc** [comment]
@mav-adhoc @augusto-weiss this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/purchase#176

**Comment by fw-bot-adhoc** [comment]
@mav-adhoc @augusto-weiss this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/purchase#176

**Comment by fw-bot-adhoc** [comment]
@mav-adhoc @augusto-weiss this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/purchase#176

**Comment by fw-bot-adhoc** [comment]
@mav-adhoc @augusto-weiss this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/purchase#176

**Comment by fw-bot-adhoc** [comment]
@mav-adhoc @augusto-weiss this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/purchase#176

**Comment by fw-bot-adhoc** [comment]
@mav-adhoc @augusto-weiss this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/purchase#176

**Comment by fw-bot-adhoc** [comment]
@mav-adhoc @augusto-weiss this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/purchase#176

**Pull request closed without merging**